### PR TITLE
Get back scroll behaviour on configure modal

### PR DESCRIPTION
**Pull request by CPatchane**
Update cozy-ui to get scroll locking + only use height for non mobile view.

### CODE REVIEW

**Comment by CPatchane** [comment]
I think this can be considered as a bug from cozy-ui, here is the related issue: https://github.com/cozy/cozy-ui/issues/668

**Comment by CPatchane** [comment]
[[STO/HOME] Je n'ai pas de scroll sur la modale de connexion d'un connecteur](https://trello.com/c/TGHkbqQV/391-sto-home-je-nai-pas-de-scroll-sur-la-modale-de-connexion-dun-connecteur)

File: src/ducks/apps/components/ConfigureModal.jsx [hunk 1]
```diff
@@ -33,7 +35,7 @@ export class ConfigureModal extends Component {
         mobileFullscreen
         overflowHidden
         size="small"
-        height="35rem"
+        height={!isMobile && '35rem'}
```
Comment (gregorylegarec) [inline src/ducks/apps/components/ConfigureModal.jsx]: Is there absolutely no way to handle this with CSS ?

Comment (CPatchane) [reply src/ducks/apps/components/ConfigureModal.jsx]: We will have to use a static media query matching the mobile breaking point and target a new class on the modal wrapper here.

Comment (gregorylegarec) [reply src/ducks/apps/components/ConfigureModal.jsx]: It does not scare me actually and I am pretty much convinced that it is what we should do.

Any opinion about this @GoOz, @ptbrowne ?

Comment (ptbrowne) [reply src/ducks/apps/components/ConfigureModal.jsx]: No I think this is OK. It is still a patch.

For me the best way for these problems to go away is for the intent to tell the app what its height ? This way the modal would automatically be the right height.

See my comment : https://github.com/cozy/cozy-ui/issues/668#issuecomment-434641325

Comment (GoOz) [reply src/ducks/apps/components/ConfigureModal.jsx]: Well it was discussed in the referenced issue.

Comment (gregorylegarec) [reply src/ducks/apps/components/ConfigureModal.jsx]: Ok then, so it’s an interapp issue and we agree on the fact that this magical 35rem value should not be here.

**Comment by gregorylegarec** [comment]
Thanks @Cpatchane 🧞‍♂️

**Pull request merged**